esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -1620,13 +1645,24 @@ bool CWsESDLConfigEx::onListESDLBindings(IEspContext &context, IEspListESDLBindi
         Owned<IEspESDLBinding> esdlbinding = createESDLBinding("","");
         esdlbinding->setId(item.queryProp("@id"));
         esdlbinding->setEspProcess(item.queryProp("@espprocess"));
+
         const char* portstr = item.queryProp("@port");
         if (portstr && *portstr)
             esdlbinding->setPort(atoi(portstr));
         else
             esdlbinding->setPort(0);
         esdlbinding->setEspBinding(item.queryProp("@espbinding"));
         list.append(*esdlbinding.getClear());
```

Comment:
Extra append line, you should remove this one because you added another a few lines down.